### ISSUE TITLE
Ads in the publications page (?)

### ISSUE THREAD

**Issue by alvarogarcia7** [codurance/site#454]
Saw this on the publications page while using AdBlock Plus:

https://idsync.weborama.fr/fcgi-bin/idsync.fcgi?d.A=rd&d.u=https%3A%2F%2Fdx.bigsea.weborama.com%2Fcollect%3Fdsp_id%3D0%26eid%3D%7BWEBO_ID%7D Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
https://www.expedia.com/ads/bkcampaign/26813?rand=1895963593 Failed to
<img width="613" alt="screen shot 2016-12-09 at 11 56 54" src="https://cloud.githubusercontent.com/assets/4199136/21048336/ae9c7fc4-be06-11e6-9fb4-3bf2eec607db.png">
 load resource: net::ERR_BLOCKED_BY_CLIENT

**Comment by neomatrix369** [comment]
Looks like an interesting issue, just out of curiosity - do a lot of people use such AdBlocking plugins that we might want to consider fixing this issue or even investigating it?

**Comment by cyplo** [comment]
I think I saw a survey somewhere that said that it is a really significant amount of traffic. Most of them seem to be using uBlock origin now, instead of AdBlock. Also iOS I think started (will start?) block ads by default in Safari.
Can dig for details if needed.

**Comment by Mattsi-Jansky** [comment]
Having poked around a bit with uBlock Origin and Privacy Badger I didn't notice any activity from Expedia or Weborama. I did notice a lot of ad activity on any video page, coming from YouTube. Other than those I saw: hs-analytics.net, google-analytics.com, leadforensics.com and sharethis.com.

The first three are all analytics. Is it perhaps worth getting rid of two of them and focusing on one? Is there a good reason why we have three? 

The last one is some kind of social widget known for tracking users. After disabling it I saw no difference in the site and all the existing social links loaded just as they had been. Perhaps it's an old tool that we don't use anymore? In a source code search it only appears to be used in `event.html`, but I'm not sure where that's used if at all.
